tests/smart/modules/test_childlock.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from kasa import Module
+from kasa.smart.modules import ChildLock
+
+from ...device_fixtures import parametrize
+
+childlock = parametrize(
+    "has child lock",
+    component_filter="button_and_led",
+    protocol_filter={"SMART.CHILD"},
```

Comment:
```suggestion
    protocol_filter={"SMART", "SMART.CHILD"},
```

This test case would not be running so presumably the coverage was addressed by the generic test all features test.

Reply:
This shouldn't appear in child devices, so I modified it manually to use only SMART.